containers/Ideas/Idea.styles.js

```diff
@@ -21,8 +21,12 @@ export const styles = StyleSheet.create({
     flex: 1,
     justifyContent: 'flex-end'
   },
+  headerContainer: {
```

Comment:
is there a way to check the statusbar and set the height dynamically?

in both it looks quite far in the middle and does not match the design. I am attaching the iphone screenshot, for the android it is slightly closer to the statusbar but still quite for imho.

<img width="439" alt="Bildschirmfoto 2022-07-14 um 17 57 04" src="https://user-images.githubusercontent.com/52801593/179025413-5df5ff7c-e955-4c07-b939-663363d7a79d.png">

Reply:
ah sorry, just saw the WIP now...